db/migrate/20151123235600_add_purchase_type_to_gsa18f_procurements.rb

```diff
@@ -0,0 +1,11 @@
+class AddPurchaseTypeToGsa18fProcurements < ActiveRecord::Migration
+  def change
+    add_column :gsa18f_procurements, :purchase_type, :integer
```

Comment:
You know, I haven't used it before! It looks like it is nice but since the behavior would be different than that standard AR enum behavior (which by now most people in the Rails world are familiar with), I am a little hesitant to introduce it here. The only good blog post I found on using postgres enum with activerecord enum is here: http://www.sitepoint.com/enumerated-types-with-activerecord-and-postgresql/

Another consideration, which I spoke with Peter about yesterday, is the behavior of enums in admin. I've dealt with crazy stuff when trying to use Rails admin with enums in the past. I am wondering if ActiveAdmin has good support for Postgres enums...